DotnetHsdpSdk/Utils/HttpRequester.cs

```diff
@@ -108,11 +110,14 @@ private static async Task<HttpResponseMessage> BuildAndExecuteRequest(IHsdpReque
         {
             Port = -1
         };
-        var query = HttpUtility.ParseQueryString(builder.Query);
-        foreach (var (key, value) in queryParameters) query[key] = value;
+        var queryBuilder = new StringBuilder();
```

Comment:
Using a string-builder is a build old-fashioned :b one can use a var query = ""; and then query += "?"; to append strings.

Reply:
Oops, pressed merge before seeing this comment. I thought string builders are more efficient?!